src/calculation/DetailedResultsBuilder.ts

```diff
@@ -151,31 +153,39 @@ export function handlePopulationValues(
   ) {
     setResult(PopulationType.DENEX, false, populationResults);
     setResult(PopulationType.DENEXCEP, false, populationResults);
-
-    if (!MeasureBundleHelpers.hasMultipleIPPs(group)) {
+    DetailedResultsHelpers.nullCriteriaRefMeasureObs(group, populationResults, PopulationType.DENOM);
+
+    // If there is a MSRPOPL, all observations point to it, so null them out
+    if (hasResult(PopulationType.MSRPOPL, populationResults)) {
```

Comment:
It looks like this branch and the MSRPOPLEX branch are not covered by tests. Can we add tests for them?